mono/metadata/profiler.c

```diff
@@ -743,6 +789,38 @@ mono_profiler_class_loaded (MonoClass *klass, int result)
 	}
 }
 
+void
+mono_profiler_static_field (MonoClassField *field)
```

Comment:
We probably could, and then emit a single event with a list of fields. I just did it this way because it fits the 'pattern' with regards to other metadata events, which is one event per metadata element.

I don't care much either way, TBH.